src/components/Transfer/components/ShownTransfer/ShownTransfer.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { FC } from 'react';
+import { ControlledLeftSide } from '../../containers/LeftSide';
+import { ControlledRightSide } from '../../containers/RightSide';
+import { InnerTransferProvider, useTransferContext } from '../../context';
+import { StyledTransfer } from '../../styles';
+import { resolveLeftSideOpen } from '../../utils/resolveLeftSideOpen';
+import { resolveRightSideOpen } from '../../utils/resolveRightSideOpen';
+
+export const ShownTransfer: FC = () => {
+    const {
+        footer,
+        customWidth,
+        preferredPosition,
+        innerTransferController,
+        positionController,
+        isOpen,
+        isFullWidth,
+        ...props
+    } = useTransferContext();
+    const { innerValue, ...innerRest } = innerTransferController(props);
+    const isLeftSideOpen = resolveLeftSideOpen(!!isOpen, innerValue);
+    const isRightSideOpen = resolveRightSideOpen(!!isOpen, innerValue);
+    const { position, refOnTransfer, leftRight } = positionController!({
```

Comment:
object literal parameter is unstable when used as a dependency, this can be dangerous if not handled properly (parameter destructuring is safe), it is better to either use multiple parameters or include a comment to destructure the parameters when trying to customize this function